kv/txn_coord_sender_test.go

```diff
@@ -270,14 +217,13 @@ func TestTxnCoordSenderMultipleTxns(t *testing.T) {
 	defer s.Stop()
 	defer teardownHeartbeats(s.Sender)
```

Comment:
while you're cleaning up, probably better to explicitly abort those two txns and to remove this defer.